src/themes/teams/components/Segment/segmentStyles.ts

```diff
@@ -14,7 +17,7 @@ const segmentStyles: ComponentSlotStylesInput<SegmentProps, SegmentVariables> =
         (p.inverted
```

Comment:
`inverted` should work even without specifying color.

Reply:
@miroslavstastny this is about the computed color not the `color` prop; there's nothing to invert if there's no color for the segment

Reply:
I understand, I was just pointing out the difference between Stardust and SUIR.
In SUIR `<Segment inverted>` is light on black. In Stardust `inverted` is ignored.

Reply:
oh got it, good point, thanks @miroslavstastny , introduced a fix for that, should work now